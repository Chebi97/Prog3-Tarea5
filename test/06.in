26
15-16-98
13-14-65
18-20-110
13-17-98
13-19-66
10-15-70
10-12-60
8-10-90
17-18-17
15-16-15
13-14-13
18-20-20
13-17-19
13-19-11
10-15-16
10-12-14
8-10-12
17-18-90
15-16-78
13-14-45
18-20-90
13-17-78
13-19-46
10-15-50
10-12-40
8-10-70
16
10-15
13-19
13-17
18-20
13-14
15-16
17-18
8-10
10-12
10-15
13-19
13-17
18-20
13-14
15-16
17-18
