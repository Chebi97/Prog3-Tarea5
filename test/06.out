Solucion Valida. asistentes: 1154. 
